fractal-terrain-generation/.gitignore

```diff
@@ -0,0 +1,63 @@
+# Logs
```

Comment:
most of these should not be here as this is now a subdir, not the main directory